x-pack/plugins/osquery/public/scheduled_query_groups/scheduled_query_group_queries_table.tsx

```diff
@@ -27,6 +26,20 @@ import { useKibana, isModifiedEvent, isLeftClickEvent } from '../common/lib/kiba
 import { PlatformIcons } from './queries/platforms';
 import { OsqueryManagerPackagePolicyInputStream } from '../../common/types';
 
+const VIEW_IN_DISCOVER = i18n.translate(
+  'xpack.osquery.scheduledQueryGroup.queriesTable.viewDiscoverResultsActionAriaLabel',
+  {
+    defaultMessage: 'View in Discover',
+  }
+);
+
+const VIEW_IN_LENS = i18n.translate(
+  'xpack.osquery.scheduledQueryGroup.queriesTable.viewLensResultsActionAriaLabel',
+  {
+    defaultMessage: 'View in Lens',
```

Comment:
@KOTungseth It is ok to say Lens here?

Reply:
Someday, `Lens` will no longer appear on the UI. To be forward thinking, this should say `Visualize in editor` or `Create visualization`.